app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<section>
+  <%= link_to "add new recipe", new_recipe_path, class: 'btn btn-success mb-3'%>
+  <ul class="list-unstyled d-flex flex-column gap-3"> 
+    <%@recipes.each do |recipe|%>
+      <li class="d-flex gap-5 border border-2 p-2">
+        <div class="w-25">
+            <%= link_to recipe_path(recipe) do%>
+              <h3 class="fs-3"><%= recipe.name%></h3>
+            <%end%>
+            <%= button_to "Remove", recipe_path(recipe.id), method: :delete, data: { confirm: "Are you sure?", turbo_confirm: "Are you sure?" }, class: 'btn btn-danger'%>
```

Comment:
- [ ] 🔐 According to the project guidelines, only the rightful owner should have the authority to delete a recipe item. It's crucial to enhance security by verifying that the user attempting the deletion is indeed the rightful owner. To achieve this, consider implementing an authorization mechanism, such as CancanCan or any other suitable solution. This mechanism will add an additional layer of protection by ensuring that only users with the appropriate permissions can access the delete functionality. This way, we strengthen the security of our application and guarantee that only authorized individuals can modify or remove recipe items. 🚀